docs/product/issues/issue-details/performance-issues/web-vitals/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Web Vitals
+sidebar_order: 30
+description: Learn more about Web Vitals issues and how to diagnose and fix them.
+og_image: >-
+  /og-images/product-issues-issue-details-performance-issues-web-vitals.png
+---
+
+Web Vitals issues are created for the highest opportunity pages in a project when
+their vitals scores fall into the [`Meh`](/product/insights/frontend/web-vitals/#performance-score) or [`Poor`](/product/insights/frontend/web-vitals/#performance-score) threshold for an extended period of time.
+
+## Detection Criteria
+
+<Alert>
+
+This issue is currently only available for browser SDKs.
+
+</Alert>
+
+The detector for this performance issue runs on a schedule and checks the 7 day p75 (75th percentile) Web Vitals scores for the top 5 [highest opportunity](/product/insights/frontend/web-vitals/#opportunity) pages on your website (_the pages that are both high traffic, and have issues worth fixing_).
```

Comment:
```suggestion
The detector for this performance issue runs on a schedule and checks the 7 day average Web Vitals scores for the top 5 [highest opportunity](/product/insights/frontend/web-vitals/#opportunity) pages on your website (_the pages that are both high traffic, and have issues worth fixing_).
```
We judge the scores based off averages. We use the p75 sample as the trace sample that gets fed to the issue/seer.